src/helpers/tokens/generate.token.js

```diff
@@ -0,0 +1,26 @@
+import jwt from 'jsonwebtoken';
+import models from '../../api/models';
+
+const { Token } = models;
+
+const createToken = (user, secret) => {
+  try {    
+    const token = jwt.sign(
+      { user },
+      secret,
+      {
+        expiresIn: '24h'
+      }
+    );
+
+    Token.create({
+      data: token,
+    });
+
+    return token;
+  } catch (error) {
+    return 'error while processing a token';
```

Comment:
@alainburindi, better to return the error message along with the error object in order to understand what went wrong.